cert_manager/Tiltfile

```diff
@@ -26,27 +26,26 @@ spec:
 """
 
 # Deploys cert manager to your environment
-def deploy_cert_manager(registry="quay.io/jetstack", version="v0.16.1", load_to_kind=False, kind_cluster_name="kind"):
+def deploy_cert_manager(registry="quay.io/jetstack", version="v1.1.0", load_to_kind=False, kind_cluster_name="kind"):
     silent=True
+    if version.startswith('v0'):
```

Comment:
This isn't super robust, might be folks from tilt can give some suggestion here...

Reply:
Yeah, I hope someone can suggest something! I have thought on the two following options:

- use the Python semver package if it is possible to import and install it with pip on Tilt
- improve it with regexes